conda_forge_tick/git_utils.py

```diff
@@ -1445,7 +1448,8 @@ def _get_pth_blob_sha_and_content(
         # so I switched to doing the decoding by hand.
         data = base64.b64decode(cnt.content.encode("utf-8")).decode("utf-8")
         return cnt.sha, data
-    except github.UnknownObjectException:
+    except (github.UnknownObjectException, github.GithubException) as e:
+        _test_and_raise_besides_file_not_exists(e)
```

Comment:
```suggestion
    except (github.GithubException) as e:
        _test_and_raise_besides_file_not_exists(e)
```

`UnknownObjectException` is a subclass of `GithubException`